### ISSUE TITLE
Settings dialog: Space not good used for language setting

### ISSUE THREAD

**Issue by HolgerJeromin** [streetcomplete/StreetComplete#5816]
**How to Reproduce**

59-alpha1

Open settings

**Expected Behavior**

![1000248049](https://github.com/user-attachments/assets/a031a65c-0509-4cab-a83a-d36a0b1581b2)

**Comment by HolgerJeromin** [comment]
With the 500px screenshot I see 230 px white space between the both words.
Would be nice if the grid could be more flexible in distributing space to the label compared to the value.

**Comment by westnordost** [comment]
Yeah, maybe, IIRC I tried that, but there didn't seem to be an easy way to do this (make it flexible)

**Comment by HolgerJeromin** [comment]
Having at least 50%/50% would be a solution.
But I know there more important parts to do right now.
